js/index.js

```diff
@@ -37,6 +37,73 @@ const siteContent = {
   },
 };
 
-// Example: Update the img src for the logo
-let logo = document.getElementById("logo-img");
-logo.setAttribute('src', siteContent["nav"]["img-src"])
+// helper functions 
```

Comment:
This is INCREDIBLE.
For you to come up with the logic and implement it with the tools you currently have is really just amazing.  

I am so impressed. Over time, as you learn more things I want you to clean this up but - considering the tools you have at this moment of time you should be proud.

In a sense the way you thought about this, breaking that object and realizing that pieces of it are just rendered out to the DOM is just so mature in thinking at this state that I can't say enough good about it.  

That thinking is going to take you far in your career, in addition to the soft-skills and ability to work with everyone.

I love this implementation and despite its complexity, the organization and commenting of it is so far beyond what I would expect in this week.

WOW.

🥇

Reply:
Thank you! 

It was very rushed today, so I'm hoping to make this much more concise. 

The DOM is such a beautiful way of getting, accessing, and manipulating data - traversing between different hierarchies - one node at a time.

I'd like to be in a place where its very possible for other programmers/designers to make modifications several months from now, without dealing with faulty documentation.